stable/kube-state-metrics/templates/podsecuritypolicy.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   privileged: false
   volumes:
     - 'secret'
+{{- if .Values.podSecurityPolicy.additionalVolumes }}
```

Comment:
Let's change it to having a `volumes parameter` and we can default it to having `secret` in the `values.yaml`
```suggestion
{{- if .Values.podSecurityPolicy.volumes }}
```

Reply:
The list should be empty here.

Reply:
We would lose backward compatibility if default it to empty

Reply:
We default to `['secret']` in `values.yml`, so the default is effectively the same afaik.